pkg/agent/config/config.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package main
+package config
```

Comment:
I think moving config from cmd to pkg is a change bigger than it needs to be. There is also config.go in cmd/antrea-controller, which would cause inconsistency when people looks for config file.

There are e2e tests that also need to get config and are dealing with text directly:
https://github.com/vmware-tanzu/antrea/blob/2634d0d4e7bd95e141055f4527327afd16b70e81/test/e2e/framework.go#L976-L977
https://github.com/vmware-tanzu/antrea/blob/2634d0d4e7bd95e141055f4527327afd16b70e81/test/e2e/security_test.go#L63
I'm not saying it's the best way but want to keep this PR focus on its issue and solve it a way consistent with existing ones and leave optimization and discussion in dedicated PR.

Reply:
Understood. Thanks, Quan.